tests/src/main/java/com/hazelcast/stabilizer/tests/map/helpers/IntegerStringEntryListener.java

```diff
@@ -26,34 +26,34 @@
     public int maxDelayMs;
 
     @SuppressWarnings("unused")
-    public EntryListenerImpl() {
+    public IntegerStringEntryListener() {
```

Comment:
is it really necessary to go round saying everything is an IntergerString...Something.  of a StringInteger..Something
Object Object  so we can reuse it,  and put any old stuff we like in ?